std/process.d

```diff
@@ -806,6 +806,30 @@ private void setCLOEXEC(int fd, bool on) nothrow @nogc
     assert (wait(spawnProcess([prog.path, "foo", "bar"])) == 0);
 }
 
+version (Posix) @system unittest
+{
+    import core.sys.posix.fcntl : open, O_RDONLY;
+    import core.sys.posix.unistd : close;
+    import std.path : buildPath;
+    import std.conv : to;
+    static import std.file;
+
+    auto directory = uniqueTempPath();
+    std.file.mkdir(directory);
+    scope(exit) std.file.rmdirRecurse(directory);
+    auto path = buildPath(directory, "tmp");
+    std.file.write(path, null);
+    auto fd = open(path.tempCString, O_RDONLY);
+    scope(exit) close(fd);
+
+    TestScript checkDefaults = `command >&0 && command >&1 && command >&2`;
```

Comment:
Sure, once I've worked out why it's failing on FreeBSD. It was about time I installed FreeBSD somewhere anyway...